### PR TITLE
Fix loki manual params doc

### PR DESCRIPTION
**Pull request by jotak**
## Description

CRD doc fixes for loki manual params (explicit urls)

## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [ ] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [ ] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [x] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

File: api/v1beta2/flowcollector_types.go [hunk 1]
```diff
@@ -521,16 +521,15 @@ type LokiManualParams struct {
...
 
-	//+kubebuilder:validation:optional
-	// `querierURL` specifies the address of the Loki querier service, in case it is different from the
-	// Loki ingester URL. If empty, the URL value is used (assuming that the Loki ingester
-	// and querier are in the same server). When using the Loki Operator, do not set it, since
-	// ingestion and queries use the Loki gateway.
+	//+kubebuilder:default:="http://loki:3100/"
+	// `querierUrl` specifies the address of the Loki querier service.
+	// When using the Loki Operator, set it to the Loki gateway service with the `network` tenant set in path, for example
+	// https://loki-gateway-http.netobserv.svc:8080/api/logs/v1/network.
```
Comment (jpinsonneau) [inline api/v1beta2/flowcollector_types.go:527]: Should we still mention loki operator in `manual` since we introduced `LokiStack` mode ?

**Review by jpinsonneau — APPROVED** [review]
Small comment, LGTM

File: api/v1beta2/flowcollector_types.go (hunk 1 above)
Comment (jotak) [reply api/v1beta2/flowcollector_types.go:527]: At first I removed them, then I thought perhaps there will be cases where people use loki operator but for some reason need to use manual params (e.g. using an old version or on the contrary a newer version which triggers changes in our lokistack mode handling) .. so perhaps it's doesn't hurt to keep this?

Comment (jpinsonneau) [reply api/v1beta2/flowcollector_types.go:527]: sure :+1:

**Review by jpinsonneau — APPROVED** [review]

**Pull request merged**